telegram_initializer.py

```diff
@@ -0,0 +1,91 @@
+import telebot
+from user import User
+import message_text_config as msg
+
+
+class TelegramInitializer:
+    def __init__(self):
+        self.token = "1608802403:AAELfG3U92U9XSQPqn5QdGxwTEZyLzULDUc"
+        self.bot = telebot.TeleBot(token=self.token)
+        self.users = []
+
+    def get_users(self):
+        @self.bot.message_handler(commands=['join'])
+        def join(message):
```

Comment:
Methods name looks like standard method. Maybe it will be better to change it to "join_player" for example.

Reply:
Done